### ISSUE TITLE
Best score for each model

### ISSUE THREAD

**Issue by melgor** [KevinMusgrave/powerful-benchmarker#10]
What value is assumes to be the best for model in your CSV file?
Is is:
 - last model?
- model which provide best score during training (ex. epoch 50/100)

I just look to [SoftTriplet](https://github.com/idstcv/SoftTriple/blob/master/train.py#L130) and they report last model.
In the other hand [Multi-Similarity Loss](https://github.com/MalongTech/research-ms-loss/blob/master/ret_benchmark/engine/trainer.py#L56) report best iteration.

We need to be clear evaluation scenario.

**Comment by KevinMusgrave** [comment]
In the "old_approach" evaluation, I report the best epoch on the test set. This goes against basic machine-learning principles, but there is no good alternative when there is no validation set. 

So I started [another spreadsheet](https://docs.google.com/spreadsheets/d/1brUBishNxmld-KLDAJewIc43A4EVZk3gY6yKe8OIKbY/edit?usp=sharing) where I do cross-validation and report the test-set performance of the best model from cross validation.
